config/index.js

```diff
@@ -28,6 +28,19 @@ var conf = convict({
       env: 'LOG_FORMAT'
     }
   },
+  memcache: {
+    address: {
+      doc: 'Hostname/IP:Port of the memcache server',
+      default: '127.0.0.1:11211',
+      env: 'MEMCACHE_ADDRESS'
+    },
+    idle: {
+      doc: 'Idle timeout for memcached connections (milliseconds)',
+      format: Number,
+      default: 30000,
```

Comment:
I meant the integration point in auth-server here: https://github.com/mozilla/fxa-auth-server/blob/master/lib/customs.js#L11

I agree it's worth running the tests against memcached though, at least in travis if not by default